sources/packages/forms/src/form-definitions/sfaa2024-25.json

```diff
@@ -22436,7 +22454,7 @@
                   "value": ""
                 }
               ],
-              "content": "Enter your reported total income from line 15000 of your 2020 income tax return. This income will be matched with Canada Revenue Agency records, which may affect your assessment of need and/or grant eligibility.  \n\nIf you did not file a 2020 income tax return, enter your total income from all sources both inside and outside of Canada.",
+              "content": "Enter your reported total income from line 15000 of your {{data.programYear}} income tax return. This income will be matched with Canada Revenue Agency records, which may affect your assessment of need and/or grant eligibility.  \n\nIf you did not file a 2020 income tax return, enter your total income from all sources both inside and outside of Canada.",
```

Comment:
Please see the below image with the questions.
![image](https://github.com/user-attachments/assets/9aac39db-3a2a-47bc-a658-bb3d9ed1f7e6)